app/customer.py

```diff
@@ -0,0 +1,31 @@
+import json
+import os
+from app.car import Car
+
+
+current_directory = os.path.dirname(os.path.abspath(__file__))
+relative_path = os.path.join(current_directory, "config.json")
```

Comment:
either move these to `customers_and_content` function or rewrite naming using uppercase: CURRENT_DIRECTORY, RELATIVE_PATH